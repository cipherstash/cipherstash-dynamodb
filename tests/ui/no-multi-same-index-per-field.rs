use cipherstash_dynamodb::Encryptable;

#[derive(Encryptable)]
struct User {
    #[partition_key]
    email: String,
    #[cipherstash(query = "exact")]
    #[cipherstash(query = "exact")]
    name: String,
}

fn main() {}
