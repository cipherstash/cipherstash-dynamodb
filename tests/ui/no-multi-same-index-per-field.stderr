error: Index 'name' with type 'exact' has been defined more than once
 --> tests/./ui/no-multi-same-index-per-field.rs:3:10
  |
3 | #[derive(Encryptable)]
  |          ^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Encryptable` (in Nightly builds, run with -Z macro-backtrace for more info)
